### DISCUSSION TITLE
Reload changes from config.py

### DISCUSSION THREAD

**Discussion by chevcheli0s** [pallets/flask#4878] [Q&A] [answered]
Is there any way to reload settings from config.py such as 

```
AUTH_ROLES_MAPPING = {
     "cn=User,ou=groups,dc=example,dc=com": ["User"],
     "cn=Admin,ou=groups,dc=example,dc=com": ["Admin"],
} 
```
without restarting app?

**Comment by tachyondecay** [comment, marked as answer]
You can call `app.config.update()` at any time to update your app's configuration, **but you probably shouldn’t**.

Remember that under most circumstances, you’ll be running multiple workers/threads and therefore multiple instances of your app. Therefore, updating the config in this way would only affect a single instance.

If you have changed your configuration, you should restart all running instances of your app.

If you have data that you expect to change while the app is running, you should store it in a database, redis, etc.—somewhere centralized that your app can query whenever it needs that data.
